content/en/docs/concepts/workloads/controllers/ttlafterfinished.md

```diff
@@ -16,7 +16,7 @@ objects that have finished execution. TTL controller only handles
 now, and may be expanded to handle other resources that will finish execution,
 such as Pods and custom resources.
 
-Alpha Disclaimer: this feature is currently alpha, and can be enabled with
+Alpha Disclaimer: this feature is currently alpha, and can be enabled with both kube-controller-manager and kube-scheduler
```

Comment:
Interesting finding. In your commit message, you said "currently, TTLafterfinished feature gate is a flag that can be passed to kube-apiserver, kube-scheduler and kube-controller-manager." Here you only mention the last two. So you see how difficult it is to remember exactly which component accepts which gate ...

Reply:
actually, the required flag here should be kube-apiserver and kube-controller-manager.